testing/execution.ts

```diff
@@ -163,10 +167,26 @@ export async function executeFormulaOrSyncFromCLI({
     if (!(syncFormula || formula)) {
       throw new Error(`Could not find a formula or sync named "${formulaName}".`);
     }
-    const formulaSpecification: SyncFormulaSpecification | StandardFormulaSpecification = {
-      type: syncFormula ? FormulaType.Sync : FormulaType.Standard,
-      formulaName,
-    };
+
+    let formulaSpecification:
+      | SyncFormulaSpecification
+      | StandardFormulaSpecification
+      | SyncMetadataFormulaSpecification;
+    if (getSchema) {
```

Comment:
Per the above comment, may just want to split this part out to a separate PR so we can iterate on executing metadata formulas more generally.